src/mSupplyMobileApp.js

```diff
@@ -68,43 +60,12 @@ import { PrescriptionPage } from './pages/PrescriptionPage';
 import { SettingsPage } from './pages/SettingsPage';
 import { DashboardPage } from './pages/DashboardPage';
 import { SupplierRequisitionPage } from './pages/SupplierRequisitionPage';
+import { navigationStyles } from './globalStyles/navigationStyles';
 
 const SYNC_INTERVAL = 10 * 60 * 1000; // 10 minutes in milliseconds.
 const AUTHENTICATION_INTERVAL = 10 * 60 * 1000; // 10 minutes in milliseconds.
 
 class MSupplyMobileAppContainer extends React.Component {
-  handleBackEvent = debounce(
```

Comment:
this is moved to be in the `navigation/actions` file, `goBack` action. Handles whether or not to go back or close the app